Src/FluentAssertions/Events/RecordedEvent.cs

```diff
@@ -12,10 +12,11 @@ internal class RecordedEvent
         /// <summary>
         /// Default constructor stores the parameters the event was raised with
         /// </summary>
-        public RecordedEvent(DateTime utcNow, params object[] parameters)
+        public RecordedEvent(DateTime utcNow, int raisedOrderIndex, params object[] parameters)
```

Comment:
Since the other occurrences of `raisedOrderIndex` was renamed to `sequence` this should be too.

Reply:
Thanks! Renamed in https://github.com/fluentassertions/fluentassertions/pull/1773/commits/d378d0ee0fbbe5923ad100ed6e3b4f3c30188579.